staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -114,29 +108,46 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 }
 
 func (s *EtcdOptions) ApplyTo(c *server.Config) error {
-	c.RESTOptionsGetter = &restOptionsFactory{options: s}
-
+	c.RESTOptionsGetter = RESTOptionsFactoryFunc(func(resource schema.GroupResource) (generic.RESTOptions, error) {
+		ret := generic.RESTOptions{
```

Comment:
If we missed this and just broke it, update the sample API server test to prove storage and actually check its location in  etcd.  We've had pain related to messing this up in openshift.